.github/workflows/pythonpackage.yml

```diff
@@ -2,14 +2,14 @@ name: Python package
 
 on: [push]
 env:
-  POETRY_VERSION: 1.1
+  POETRY_VERSION: 1.1.15
 
 jobs:
   testing:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        PYTHON_VERSION: ["3.6", "3.7"]
+        PYTHON_VERSION: ["3.9.16"]
```

Comment:
But then we aren't testing support for 3.6 or 3.7.  I know 3.6 is end of life already, and 3.7 in a few months, but if folks who rely on this library are on 3.6, then we really should be testing it!  And if we want to move on from 3.6, we need to do so [here](https://github.com/fulcrumgenomics/pybedlite/blob/a2660a452fe38794324996ebe3bee6fc4b665f91/pyproject.toml#L28) and make a big announcement.

Reply:
@nils please see this commit https://github.com/fulcrumgenomics/pybedlite/pull/11/commits/5b6c30fd981c400e3162a6bc0eba7659d5ea0b3b specifying ubuntu-20.4 allows for testing of 3.6 and 3.7 :)